data/sources/unassigned.json

```diff
@@ -1,6 +1,5 @@
 {
   "164083100": "Source: Display of Supremacy, Weekly Challenge",
-  "2939318890": "Source: Wayfinder's Compass",
   "3094114967": "Source: Season of the Lost ritual playlists",
   "3100439379": "Source: Mission Exorcism",
```

Comment:
not sure what this is yet.

Reply:
if you mean the Exorcism line, also confusing

Reply:
yes